moja-kuhinjica-frontend/src/components/labels/RegularLabel.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { RegularLabelStyled } from './style'
+
+const RegularLabel = (props: any) => {
```

Comment:
If you are already using typescript then you should define prop types. Also you should deconstruct them

Reply:
I made interfaces that I use as prop types.